config/jest.config.js

```diff
@@ -39,6 +39,6 @@ module.exports = {
       'src',
     ],
     transformIgnorePatterns: [
-      'node_modules/(?!(@guardian/src-foundations|@guardian/types|@guardian/src-icons)/)',
+      'node_modules/(?!(@guardian/src-foundations|@guardian/types|@guardian/src-icons|@guardian/image-rendering)/)',
```

Comment:
This seems like an array already, would it make sense to use it that way instead of an ever-growing regex?

Reply:
That sound sensible. At the moment this says "ignore `node_modules` except for these four libraries". Is there a way to achieve that with an array?

*Side note* I think ideally we'd say "ignore `node_modules` except for `@guardian/*`".

Reply:
Ah yes of course! I was reading it the other way around.

I think that your suggestion is [easy to implement](https://regex101.com/r/h1ZXGw/3):

```regex
node_modules/(?!@guardian)
```